locale/enUS.lua

```diff
@@ -154,5 +154,13 @@ L.HIDE_OWN_PROFILE = "Hide Personal RaiderIO Profile Tooltip"
 L.HIDE_OWN_PROFILE_DESC = "When set this will not show your own RaiderIO Profile Tooltip, but may show other player's if they have one."
 L.SHOW_RAID_ENCOUNTERS_IN_PROFILE = "Show Raid Encounters in Profile Tooltip"
 L.SHOW_RAID_ENCOUNTERS_IN_PROFILE_DESC = "When set this will show Raid Encounter progress in RaiderIO Profile Tooltips"
+L.LEADERBOARD_TITLE = "M+ Dungeon Leaderboards"
+L.LEADERBOARD_STATUS_FULL = "Full"
+L.LEADERBOARD_STATUS_ALMOST_FULL = "Almost Full"
+L.LEADERBOARD_STATUS_OPEN = "Open"
+L.LEADERBOARD_MINIMUM = "Party minimum keystone"
```

Comment:
Not sure about this string, don't know what you think @jahraphael 

Context: This is a level required to be on the leaderboard.

(eg, here https://file.wisak.me/f/cV8oXl9qEJ.png I need to do a 8 to be on the leaderboard.)